src/ImageSharp/Formats/Png/PngOptimizeMethod.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+
+namespace SixLabors.ImageSharp.Formats.Png
+{
+    /// <summary>
+    /// Provides enumeration of available PNG optimization methods.
+    /// </summary>
+    [Flags]
+    public enum PngOptimizeMethod
```

Comment:
As I mentioned on gitter, it's better if our naming should reflect functional semantics rather than a specific use case (optimization).  In that case "positive" flags may be easier than negative ones expressing the suppression.
Suggestion:
```csharp
public enum PngChunkFilter
{
    IncludeNone = 0,
    IncludePhysicalChunk = 1 << 0,
    IncludeGammaChunk = 1 << 1,
    IncludeExifChunk = 1 << 2,
    IncludeTextChunks = 1 << 3,

    IncludeAll = IncludePhysicalChunk | IncludeGammaChunk | IncludeExifChunk | IncludeTextChunks
}
```

Reply:
I much prefer the suggested  naming definition.

The overall metadata API is designed to preserve as much metadata as possible so I think we should adopt the negative flags. This to me as a consumer would cognitively reflect that intention as I'm building a rule for what to exclude.

Reply:
this is changed now to negative flags